examples/with-ledger/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@web3-onboard/ledger": "^2.1.6",
-    "@web3-onboard/react": "^2.4.1",
+    "@web3-onboard/react": "^2.6.7-alpha.2",
```

Comment:
Just an FYI we have come across - if using the example packages for testing prior to an alpha version being published they may return a false positive as yarn workspaces doesn't behave the same as a published package